packages/storybook/stories/Breadcrumbs.stories.jsx

```diff
@@ -29,8 +29,8 @@ const Template = args => (
 
 const defaultArgs = {};
 
-export const Default = Template.bind({});
+export const Default = Template.bind(null);
 Default.args = { ...defaultArgs };
 
-export const MobileFirst = Template.bind({});
+export const MobileFirst = Template.bind(null);
```

Comment:
The storysource addon incorrectly displays the story when there are multiple stories bound to the same template and passing in `{}`. Updating them to null correctly displays the story in the Canvas Story tab. Although this works, the highlighting provided by the addon does not work properly. If we want users to be able to select a story through Canvas Story and have the addon automatically highlight the specific story template, then I believe we'll need to create templates for each story to support this functionality.

Reply:
How it is now:

https://user-images.githubusercontent.com/36863582/171962436-ed86ed45-af41-4483-af29-ccbcc86e0474.mov

When using unique templates for each story:

https://user-images.githubusercontent.com/36863582/171962488-297324f2-dde0-4ece-87c9-92b14d775528.mov